Alert/alert.css

```diff
@@ -0,0 +1,133 @@
+:root{
+    --primary-color:#1d4ed8;
+    --text-color:#e5e7eb;
+    --secondary-color:#9ca3af;
+    --bg-color:#e5e7eb;
+}
+
+.home-header {
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    background-color: var(--primary-color);
+    color: var(--text-color);
+    border-radius: 10px;
+  }
+  
+  .brand {
+    flex: 1;
+    margin-left: 1rem;
+    font-weight: 400;
+  }
+  
+  .nav-links {
+    display: flex;
+    list-style: none;
+    flex: 1;
+    justify-content: space-around;
+  }
```

Comment:
the class-name could be something more specific like nav-group/ menu. It's too similar to nav-link